docs/configuration/downstream/koji_build.md

```diff
@@ -77,8 +87,16 @@ Packit dist-git configuration.
      (don't forget to put the group name in quotes - a token in YAML is not allowed to start with `@`)
    - `all_admins` alias - allowing all users with admin access to the dist-git repo
    - `all_committers` alias - allowing all users with commit access to the dist-git repo
+* **sidetag_group** - name of a sidetag group this Koji build should be tagged into.
+  The name has to be unique across all projects that use Packit, so don't hesitate to be descriptive.
```

Comment:
Ah, that's interesting. A few questions unrelated to docs:
* Can we provide a list of Packit's used side-tags to check / get inspired? It's being removed, right?
* Can this be somehow misused?
* Can't we provide a default or suggestions? (Like `{package1}-{package2}-releases`)?

Reply:
> * Can we provide a list of Packit's used side-tags to check / get inspired? It's being removed, right?

Sidetags or sidetag groups? I assume you mean sidetag groups, as users (Packit included) have no control over sidetag naming. Well, we could provide a list, but there is only one group so far, `packit-specfile-ogr-releases`. Sidetag groups are not removed (they could be removed as part of https://github.com/packit/packit-service/issues/2412, but that's not implemented yet), sidetags are.

> * Can this be somehow misused?

I suppose so, you could tag "unwanted" builds into someone else's sidetags or untag someone else's builds, but you can do the same with `koji tag-build/untag-build` - there are no sidetag permissions, any packager is allowed tag/untag anything into/from a sidetag.

> * Can't we provide a default or suggestions? (Like `{package1}-{package2}-releases`)?

Sure, if you think this format/convention makes sense then I'll include it as a suggestion.

Reply:
> Sure, if you think this format/convention makes sense then I'll include it as a suggestion.

I've added `${package1}-${package2}...${packageN}-updates`, I think `updates` is more fitting from downstream maintainer PoV, but I can change it to `releases`.